From 86095d16f7aa7b60b231271d2eb4b4b6e2990aea Mon Sep 17 00:00:00 2001
From: Dinolek <git@dinolek.me>
Date: Thu, 19 Jun 2025 21:19:31 +0200
Subject: [PATCH 029/115] dt-bindings: memory-controllers: mediatek,smi-larb:
 Add support for MT6785

Add mediatek,mt6785-smi-larb compatible.

Signed-off-by: Dinolek <git@dinolek.me>
---
 .../bindings/memory-controllers/mediatek,smi-larb.yaml         | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/Documentation/devicetree/bindings/memory-controllers/mediatek,smi-larb.yaml b/Documentation/devicetree/bindings/memory-controllers/mediatek,smi-larb.yaml
index 2e7fac4b5..69c4feed0 100644
--- a/Documentation/devicetree/bindings/memory-controllers/mediatek,smi-larb.yaml
+++ b/Documentation/devicetree/bindings/memory-controllers/mediatek,smi-larb.yaml
@@ -20,6 +20,7 @@ properties:
           - mediatek,mt2701-smi-larb
           - mediatek,mt2712-smi-larb
           - mediatek,mt6779-smi-larb
+          - mediatek,mt6785-smi-larb
           - mediatek,mt6795-smi-larb
           - mediatek,mt6893-smi-larb
           - mediatek,mt8167-smi-larb
@@ -82,6 +83,7 @@ allOf:
       properties:
         compatible:
           enum:
+            - mediatek,mt6785-smi-larb
             - mediatek,mt8183-smi-larb
             - mediatek,mt8186-smi-larb
             - mediatek,mt8188-smi-larb
@@ -117,6 +119,7 @@ allOf:
               - mediatek,mt2701-smi-larb
               - mediatek,mt2712-smi-larb
               - mediatek,mt6779-smi-larb
+              - mediatek,mt6785-smi-larb
               - mediatek,mt8186-smi-larb
               - mediatek,mt8188-smi-larb
               - mediatek,mt8192-smi-larb
-- 
2.56.0

